### PR TITLE
[APB-1037][LS,PT] Add url to agent mapping frontend

### PR DESCRIPTION
**Pull request by Smudgey**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -56,6 +56,11 @@ microservice {
       external-url = "http://localhost:9401"
     }
 
+    agent-mapping-frontend {
```
Comment (frankoid) [inline conf/application.conf]: You have nailed the pattern 👍

File: test/uk/gov/hmrc/agentservicesaccount/config/ExternalUrlsSpec.scala [hunk 2]
```diff
@@ -35,7 +35,11 @@ class ExternalUrlsSpec extends UnitSpec with ResettingMockitoSugar {
   val completeGgSignInUrl = s"$companyAuthFrontendExternalBaseUrl$ggSignInPath?continue=${urlEncode(externalUrl + routes.AgentServicesController.root())}"
   val completeGgSignOutUrl = s"$companyAuthFrontendExternalBaseUrl$ggSignOutPath?continue=${urlEncode(ggSignOutContinueUrl)}"
 
-  val configuration = resettingMock[Configuration]
+  val mappingExternalUrl = "http://www.example.com/foo"
```
Comment (frankoid) [inline test/uk/gov/hmrc/agentservicesaccount/config/ExternalUrlsSpec.scala]: `mappingExternalUrl` is a base URL so won't have a path - `val mappingExternalUrl = "http://www.example.com"` would be more realistic

File: test/uk/gov/hmrc/agentservicesaccount/controllers/AgentServicesControllerSpec.scala [hunk 3]
```diff
@@ -89,6 +88,9 @@ class AgentServicesControllerSpec extends WordSpec with Matchers with OptionValu
       content should not include "Agent Services Account"
       content should include(messagesApi("agent.services.account.heading"))
       content should include(messagesApi("agent.services.account.heading.summary"))
+      content should include(messagesApi("agent.services.account.additional.links.title"))
+      content should include(messagesApi("agent.services.account.additional.links.mapping.body"))
+      content should include(messagesApi("agent.services.account.additional.links.mapping.url"))
       content should include(arn)
       content should include(signOutUrl)
```
Comment (frankoid) [inline test/uk/gov/hmrc/agentservicesaccount/controllers/AgentServicesControllerSpec.scala]: could add a test that the mapping URL itself is shown on the page (similar to this one for `signOutUrl`)

Comment (Smudgey) [inline test/uk/gov/hmrc/agentservicesaccount/controllers/AgentServicesControllerSpec.scala]: Added your feedback to: https://github.com/hmrc/agent-services-account-frontend/pull/16

**Pull request merged**